app/src/main/java/com/sunasterisk/youtubebackground/data/model/Video.kt

```diff
@@ -0,0 +1,22 @@
+package com.sunasterisk.youtubebackground.data.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+
+@Parcelize
+data class Item(
```

Comment:
```suggestion
data class Video(
```
Đặt tên model rõ ràng